File: cifrado/web/codigo/Cifrados C#/clases/Texto/Texto.cs
using System;
using System.Collections;

namespace Cifrado_cesar_poo_desktop.Texto
{
    public interface Texto
    {
        int numero_caracteres();
        String obtener_texto();
        void agregar_grupos_caracteres(String grupo_caracteres);
        char caracter_en_posicion(int posicion);
        int posicion_de_caracter(char caracter);
        bool hay_caracter_en_posicion(int posicion);
        void agregar_caracter(char caracter);
        void eliminar_ultimo_caracter();
        ArrayList texto_en_grupos(int grupos);
        void agregar_texto(String texto);

    }
}

File: cifrado/escritorio/codigo/Cifrados Kotlin/src/Texto/Texto.kt
package Texto

interface Texto {
    fun numero_caracteres():Int;
    fun obtener_texto():String;
    fun agregar_grupos_caracteres(grupo_caracteres:String);
    fun caracter_en_posicion(posicion:Int):Char;
    fun posicion_de_caracter(caracter:Char):Int;
    fun hay_caracter_en_posicion(posicion:Int):Boolean;
    fun agregar_caracter(caracter:Char);
    fun eliminar_ultimo_caracter();
    fun texto_en_grupos(grupos:Int):ArrayList<String>;

}






